helix-term/src/commands.rs

```diff
@@ -333,8 +338,18 @@ impl Command {
         surround_add, "Surround add",
         surround_replace, "Surround replace",
         surround_delete, "Surround delete",
-        select_textobject_around, "Select around object",
-        select_textobject_inner, "Select inside object",
+        select_textobject_around_word, "Word",
```

Comment:
I think it would be easier to review if the other textobject changes were a separate PR (I quite like it btw, the infobox works with it too !)

Reply:
Yeah, I didn't separate it out because I am lazy to split it out.